src/sentry/models/release.py

```diff
@@ -465,6 +465,13 @@ class Meta:
             ("organization", "date_added"),
             ("organization", "status"),
         )
+        indexes = [
+            models.Index(
+                fields=["organization", "version"],
+                opclasses=["", "text_pattern_ops"],
+                name="version_btree_idx",
```

Comment:
```suggestion
                name="sentry_release_version_btree_idx",
```

Reply:
@beez I renamed it to `sentry_release_version_btree` because there is a 30 char limit

Reply:
I see you had an issue with the length of the index name but if the migration and model index names won't match on change or removal the generated SQL will again use the model index name.

Reply:
Ah right, this should be fixed now!
Do I need to merge this before we run the migration or after?

Reply:
You can merge now, indexes are safe here and won't cause problems if they don't exist.